src/vs/workbench/parts/files/electron-browser/fileActions.contribution.ts

```diff
@@ -334,7 +334,7 @@ MenuRegistry.appendMenuItem(MenuId.ExplorerContext, {
 	group: 'navigation',
 	order: 20,
 	command: revealInOsCommand,
-	when: ResourceContextKey.HasResource
+	when: ResourceContextKey.Scheme.isEqualTo('file')
```

Comment:
@isidorn I think we can only ever reveal files

Reply:
@bpasero yes